clients/venice-push-job/src/test/java/com/linkedin/venice/hadoop/VenicePushJobTest.java

```diff
@@ -118,27 +118,6 @@ public void testRepushTTLJobWithBatchStore() {
     pushJob.run();
   }
 
-  @Test(expectedExceptions = VeniceException.class, expectedExceptionsMessageRegExp = ".*Repush TTL is not supported when the store has write compute enabled.*")
```

Comment:
Do we have a test case for the happy path?

Reply:
No. I can add one

Reply:
Added.